challenge_spr/README.md

```diff
@@ -6,46 +6,31 @@ Launch procedure:
 ```
 amigo-start
 amigo-free-mode
-rosrun challenge_spr challenge_spr.py amigo
+amigo-challenge_spr
 ```
 
 Testing the detection individually:
 ```
-rosrun challenge_spr detect.py amigo
+amigo-start
+amigo-free-mode
+./detect.py amigo
```

Comment:
use rosrun

Reply:
There is a amigo-challenge-spr alias, that should be the thing to use.

Reply:
But not for testing